### PR TITLE
[XHAB-81] updates home screen design

### PR DESCRIPTION
**Pull request by vpaniush**
mob1
mob2
web1
web2

### CODE REVIEW

File: lib/src/presentation/scenes/habit/habit_row.dart [hunk 1]
```diff
@@ -122,9 +125,9 @@ class _HabitRowState extends State<HabitRow> {
         width: _screenSize.width * 0.5,
         margin: EdgeInsets.only(left: _screenSize.width * 0.1),
         padding: EdgeInsets.only(
-            left: _screenSize.width > 1100
+            left: _screenSize.width > 1150
```
Comment (bogdanustyak) [inline lib/src/presentation/scenes/habit/habit_row.dart]: @vpaniush would be great to extract these magic numbers into constants

File: lib/src/presentation/scenes/home/home_screen.dart [hunk 2]
```diff
@@ -158,17 +162,24 @@ class _HomeScreenState extends State<HomeScreen> {
                             Text(
                               daysWords[weekDays[index].weekday],
                               style: TextStyle(
-                                  fontSize: _screenSize.shortestSide * 0.024),
+                                fontSize: _screenSize.shortestSide * 0.024,
+                                color: Color(0xffc8c8c8),
```
Comment (bogdanustyak) [inline lib/src/presentation/scenes/home/home_screen.dart]: @vpaniush naming for colors as well

**Review by bogdanustyak — CHANGES_REQUESTED** [review]

File: lib/src/presentation/scenes/habit/habit_row.dart (hunk 1 above)
Comment (vpaniush) [reply lib/src/presentation/scenes/habit/habit_row.dart]: @bogdanustyak updated

File: lib/src/presentation/scenes/home/home_screen.dart (hunk 2 above)
Comment (vpaniush) [reply lib/src/presentation/scenes/home/home_screen.dart]: @bogdanustyak updated

**Review by bogdanustyak — APPROVED** [review]

**Pull request merged**